matlab/schemas/+experiment/Projector.m

```diff
@@ -0,0 +1,15 @@
+%{
+experiment.Projector (lookup) # 
```

Comment:
how come this first line does not crash, is it assumed that the first line will be a comment? (can't find docs on https://docs.datajoint.io/matlab/definition/03-Table-Definition.html). if it is a comment it needs to be better than `experiment.Projector(lookup)` that doesn't add any information

Reply:
it turned out that the way i did was an outdated style of defining tables. It still works but i changed so that it meets the latest style. fixed